virtual_workstation_setup.sh

```diff
@@ -39,43 +45,53 @@ evg_user() {
 # arg3: block to inject
 # arg4: if non empty, do not append if the file does not exist
```

Comment:
Please add a note here saying the function should only be used to load files in this repo. We shouldn't be adding non-trivial logic to "global" files outside of the repo which could potentially interfere with other EVG projects' setup scripts

Reply:
Technically we're already doing that, given that we're modifying the bash profile/rc. Unless i'm misinterpreting what you're asking for.